pkg/cel/policies/mpol/engine/engine.go

```diff
@@ -180,33 +207,59 @@ func (e *engineImpl) handlePolicy(ctx context.Context, mpol Policy, attr admissi
 }
 
 func (e *engineImpl) MatchedMutateExistingPolicies(ctx context.Context, request engine.EngineRequest) []string {
```

Comment:
Why do we care about whether it's a mutate existing policy or not? For JSON payloads, it will be evaluated via CLI and there's no concept of admission/existing resources.